internal/db/storage/blob/cloudstorage/cloudstorage.go

```diff
@@ -0,0 +1,80 @@
+package cloudstorage
+
+import (
+	"context"
+	"io"
+
+	iblob "github.com/vdaas/vald/internal/db/storage/blob"
+	"github.com/vdaas/vald/internal/db/storage/blob/cloudstorage/reader"
+	"github.com/vdaas/vald/internal/db/storage/blob/cloudstorage/writer"
+	"github.com/vdaas/vald/internal/errors"
+	"gocloud.dev/blob"
+)
+
+type client struct {
+	urlstr string
+
+	bucket *blob.Bucket
+
+	readerOpts *blob.ReaderOptions
+	writerOpts *blob.WriterOptions
+}
+
+// New returns blob.Bucket implementation.
+func New(opts ...Option) (iblob.Bucket, error) {
+	c := new(client)
+
+	for _, opt := range append(defaultOpts, opts...) {
+		if err := opt(c); err != nil {
+			return nil, err
+		}
+	}
+	return c, nil
+}
+
+func (c *client) Open(ctx context.Context) (err error) {
+	c.bucket, err = blob.OpenBucket(ctx, c.urlstr)
```

Comment:
![badge](https://img.shields.io/badge/review-must-red.svg?style=flat-square)
Please use the `OpenBucketURL` function in gcsblob package instead.
https://godoc.org/gocloud.dev/blob/gcsblob#URLOpener.OpenBucketURL

Reply:
Thank you for your review.
I will fix it soon